examples/code/fcm/query_handler.mlt

```diff
@@ -206,13 +206,16 @@ val dispatch :
   Sexp.t -> Sexp.t Or_error.t = <fun>
 |}];;
 [@@@part "13"];;
+open Stdio;;
 let rec cli dispatch_table =
   printf ">>> %!";
   let result =
-    match In_channel.input_line stdin with
+    match In_channel.input_line In_channel.stdin with
```

Comment:
`match In_channel.(input_line stdin)` instead?

Reply:
A good idea. I'll do that.